packages/std/src/math/decimal.rs

```diff
@@ -2002,6 +2050,42 @@ mod tests {
         ));
     }
 
+    #[test]
+    fn decimal_to_uint_floor_works() {
```

Comment:
Are those tests even needed? You have them in examples which makes them automatically tests (unless we disabled this behaviour for some reason? I don't think so). To me it is just a duplicate.

Reply:
Yeah, some are duplicates because I wrote the examples first. But I think it's good to have both to be able to optimize examples for teaching and tests for coverage long term.

> You have them in examples which makes them automatically tests (unless we disabled this behaviour for some reason? I don't think so).

Yeah, doc tests are running as usual here